### ISSUE TITLE
o-forms: low contrast on focus states - ID: DAC_Pseudo_Radio_Links_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#532]
# Issue
This is from page 90 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=pseudo-radio-links&system_code=origami-repo-data&brand=master#>

The focus indicator for ‘Link A’ fails to meet WCAG AA guidelines for non-text contrast. This issue may affect low vision users who may find difficulty in distinguishing that focus has been placed on the link. To meet WCAG AA guidelines for non-text contrast, the minimum ratio required is 3:1 (it is currently 1.3:1).

 
# Proposed Solution:
Ensure the border colour meets or exceeds the minimum required contrast ratio of 3.1 against adjacent colours in order to satisfy WCAG 2.1 Level AA criteria. This will enable low- vision and keyboard only users to be able to identify where the focus is on the page correctly.

Additionally, emulating links with radio buttons maybe confusing for users of assistive technologies and should be given an appropriate role of radio or button as well as a shared label using a fieldset and legend. Further information can be found in the Radios section of the GOV.UK Design System.

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=checkboxes&system_code=origami-repo-data&brand=master> and found on page 92 of the audit report

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=radio-box&system_code=origami-repo-data&brand=master> and found on page 110 of the audit report

**Comment by notlee** [comment]
Similar to o-buttons: https://github.com/Financial-Times/origami/issues/504#issuecomment-1225899017

May be tricky without a breaking change. We probably want to update all focus states along with this. Big.

**Comment by notlee** [comment]
https://github.com/Financial-Times/origami/pull/842
